Libraries/src/Amazon.Lambda.AspNetCoreServer/AbstractAspNetCoreFunction.cs

```diff
@@ -495,7 +495,7 @@ protected async Task<TRESPONSE> ProcessRequest(ILambdaContext lambdaContext, obj
                     }
 
                     _logger.LogError(sb.ToString());
-                    defaultStatusCode = 500;
+                    ((IHttpResponseFeature)features).StatusCode = 500;
```

Comment:
shouldn't it work without casting?

https://github.com/aws/aws-lambda-dotnet/blob/49c1d23754cdb7896664107f46c1e88990b7f060/Libraries/src/Amazon.Lambda.AspNetCoreServer/Internal/InvokeFeatures.cs#L27

Reply:
No because InvokeFeatures implements the interfaces using explicitly interface implementation. It requires you to access the properties and methods of the interface by the variable actually being declared or casted as the interface.